### DISCUSSION TITLE
Version 2.5

### DISCUSSION THREAD

**Discussion by derschelli** [MiczFlor/RPi-Jukebox-RFID#2053] [General]
Hello,

at the moment i am building a second phoniebox and i want to sycn both boxes.

i found this:
"This feature is integrated since version 2.5" on the following page: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Sychnronising-Phonieboxes-in-a-local-network

i just did a clean installation on a new sd card and ended up on version 2.4 with no sync-feature. is 2.5 not yet released? the page describing the syncronisation was last edited in march?!

can i switch to version 2.5 after doing a fresh installation? how is that done?

thank you!

**Comment by AlvinSchiller** [comment]
Hi, 
Version 2.5 is not yet released (See https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2029).

To use this feature you can install the pre-release branch "develop" (see https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/INSTALL-stretch#b-developer-install-script-for-buster).

Hope this helps

**Reply by derschelli** [reply]
Thank you. I tried that before and was confused because under "Info" it still says Version "2.4 - 7f086efe - develop". 🙈🙈🙈

**Reply by Mannshoch** [reply]
@AlvinSchiller I'm a bit confused by the currend Versions around. There is 2.4 stable form last year and 3.2.1 as beta also from last year. Reading here it seems a version 2.5 comes around.

If Idont wish to use the stable version, yould you recommend to use 2.5 or 3.2.1 ?
What's the cause for developing on 2.x **and** 3.x ?

**Reply by s-martin** [reply]
2.4 is the current release (a bit dated) -> branch master

2.5 is the next release (not yet released) -> branch develop

3.x is a complete rewrite completely in Python, I would consider it "stable beta", but some features are missing -> branches future3/* and all issues labeled with future3

Hope that explains it a little bit.

**Reply by Mannshoch** [reply]
Why not focus solely on 3.x?

**Reply by Mannshoch** [reply]
The question came up because 2.5 appears to offer new features and is not just a maintenance update.

**Reply by s-martin** [reply]
Mainly because there was no "formal decision" yet, if we put 2.x in a strict maintenance mode and just focus on 3.x.

Some features are still missing, check here https://rpi-jukebox-rfid.readthedocs.io/en/latest/featurelist.html

In addition we currently don't have an upgrade path from 2.x to 3.x

**Reply by Mannshoch** [reply]
OK, thanks a lot for claryfication.

**Reply by AlvinSchiller** [reply]
As the discussion was originally about it: The sync feature added in 2.x is also available for 3.x on future3/develop
